app/routes/product-details.$productSlug/product-details.module.scss

```diff
@@ -0,0 +1,51 @@
+.page {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    gap: 32px;
+    padding: 3% 0;
```

Comment:
Is vertical padding supposed to be based on page width?

Reply:
Took it from the original website